### DISCUSSION TITLE
Spleen 3D Segmentation Tutorial

### DISCUSSION THREAD

**Discussion by agokli** [Project-MONAI/MONAI#8259] [Q&A]
I downloaded data (Task009_spleen.tar)  and looked into implementation (Tutorial - Spleen 3D Segmentation). 

Question - Why test - Nifti files (ImageTs folder) are not used as part of test data. This is usual practice to verify model in independent manner.

**Comment by KumoLiu** [comment]
Hi @agokli, thanks for the interest here.

I didn't get your point.
"imagesTs" has been used in the "Inference on Test Set" section in the [notebook](https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb).

Thanks.
